src/open_clip/transformer.py

```diff
@@ -255,6 +295,8 @@ def __init__(
         scale = width ** -0.5
         self.class_embedding = nn.Parameter(scale * torch.randn(width))
         self.positional_embedding = nn.Parameter(scale * torch.randn(self.grid_size[0] * self.grid_size[1] + 1, width))
+        self.patch_dropout = PatchDropout(patch_dropout)
```

Comment:
let's add a comment here saying that if patch_dropout is 0 then this does nothing